### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by vikivivi**
Co-authored-by: Peter Newman <peternewman@users.noreply.github.com>

### CODE REVIEW

**Comment by peternewman** [comment]
While this may look neater, it now means I have to review the whole thing again or painstakingly look at #1522 to work out what's new and if you fixed all my previous comments.

If we were really worried we could squash or rebase when we merge the PR.

**Comment by vikivivi** [comment]
Pardon my ignorance in using github code review GUI editing. After my editing, I felt PR #1522 is too messy to continue reviewing.

> bootlaoder->bootloader
I added plural version.

> certifiate->certificate
Your Review: Can you add the -ed, -ion, -es and -ing variants of this typo please.
It is done.

> credetial->credential
Your Review: Can you add the plural version of this typo too please.
It is done

> exipration->expiration
Your review: Can you add these too:
+ exipre->expire
+ exipred->expired
As suggested, it is done.

> temeprature->temperature
Your review: Can you add the plural please.
It is done.

Thank you.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7080,7 +7090,7 @@ cursot->cursor
 cursro->cursor
 curvelinear->curvilinear
 customable->customizable
-custome->custom, costume,
+custome->custom, costume, customer,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:7093]: Add customs as a suggestion

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -10720,6 +10730,9 @@ exibition->exhibition
 exibitions->exhibitions
 exicting->exciting
 exinct->extinct
+exipration->expiration
+exipre->expire
+exipred->expired
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:10735]: Add expires typo
```suggestion
exipred->expired
exipres->expires
```

**Comment by peternewman** [comment]
> Pardon my ignorance in using github code review GUI editing. After my editing, I felt PR #1522 is too messy to continue reviewing.

Don't worry, your certainly not the only one!

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (vikivivi) [reply codespell_lib/data/dictionary.txt]: Update suggestion: custome->custom, customs, costume, customer,

**Pull request merged**